app/experimenter/experiments/api/v3/serializers.py

```diff
@@ -30,6 +30,14 @@ class Meta:
             "audience",
         )
 
+    def validate_name(self, name):
+        slug = slugify(name)
+        if slug:
+            return name
+        raise serializers.ValidationError(
+            ["Name needs to contains alphanumeric characters"]
```

Comment:
nit: Should this be "contain" singular? (here and then in the assert below)

Reply:
gah, bad grammar 😅